### DISCUSSION TITLE
RDF-star doesn't seem to be working

### DISCUSSION THREAD

**Discussion by abecciu** [oxigraph/oxigraph#754] [Q&A] [answered]
I'm trying to import a RDF-star graph expressed in turtle-star syntax. But I consistently get 400 error whenever I POST to `/store` in the oxigraph server.

Here's the request that I'm sending:
`curl -v -X POST -H 'Content-Type:application/turtle' -T example-turtle-star.ttl http://127.0.0.1:7878/store`

The response I get is:
```
< HTTP/1.1 400 Bad Request
< content-type: text/plain; charset=utf-8
< server: Oxigraph/0.3.22
< content-length: 46
* The requested URL returned error: 400
* Closing connection
```

The content of the file that I'm trying to import is the following:
```
PREFIX : <http://example.org/>

# Standard RDF triple
:John :hasMother :Jane.

# RDF-star triple (a triple about a triple)
<< :John :hasMother :Jane >> :accordingTo :Witness1.

# Another RDF-star triple with a literal value and datatype
<< :John :birthDate "2000-01-01"^^xsd:date >> :accordingTo :BirthCertificate.

# RDF-star triple with a blank node
_:b0 :type :Review.
<< :Book1 :hasReview _:b0 >> :accordingTo :Reviewer2.
```

Do I need to do anything especial to get rdf-star to work? Thanks

**Comment by Tpt** [comment, marked as answer]
Hey! Thank you for using Oxigraph. It's weird that `curl` does not prints the content of the HTTP response. When I try to load your file on the currently unreleased `main` branch I get the output:
```
Parser error between at line 10 between columns 35 and column 43: The prefix xsd: has not been declared
```
This can be fixed by adding the following line at the beginning of the file
```turtle
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
```
And then it seems to work.

May you try again by adding the same line as me?

**Reply by abecciu** [reply]
Thanks so much for the quick response. That worked. I had an extra flag in the curl command that suppressed the request body.
